src/interfaces/IKintoAppRegistry.sol

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+interface IKintoAppRegistry {
+    /* ============ Structs ============ */
+
+    struct Metadata {
+        address developerWallet; // 160 bits
+        bool dsaEnabled;
+        uint256 rateLimitPeriod;
```

Comment:
what's the diff between rateLimitNumber and rateLimitPeriod?

Reply:
i think i got it, but just to confirm:
- `rateLimitPeriod` -> period in which the given user can execute X amount of user operations
- `rateLimitNumber` -> max amount of user operations that a user can execute PER PERIOD

if period is 5 hours and number is 10, a user can execute max 10 user operations every 5 hours, right?

Reply:
correct